docs/install/ipfs-desktop.md

```diff
@@ -13,16 +13,16 @@ If you already have an IPFS node on your computer, IPFS Desktop will act as a co
 
 ![Status screen of IPFS Desktop](./images/ipfs-desktop/desktop-status.png)
 
-| Files screen                                                               | Explore screen                                                                 | Peers screen                                                               | Settings screen                                                                  | Menubar/taskbar                                                                       |
+| Files screen                                                               | Explore screen                                                                 | Peers screen                                                               | Settings screen                                                                  | Menubar: top right                                                                      |
 | -------------------------------------------------------------------------- | ------------------------------------------------------------------------------ | -------------------------------------------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
 | ![Screenshot of the Files screen](./images/ipfs-desktop/desktop-files.png) | ![Screenshot of the Explore screen](./images/ipfs-desktop/desktop-explore.png) | ![Screenshot of the Peers screen](./images/ipfs-desktop/desktop-peers.png) | ![Screenshot of the Settings screen](./images/ipfs-desktop/desktop-settings.png) | ![Screenshot of Mac/Windows menus](./images/ipfs-desktop/desktop-menubar-taskbar.png) |
 
 ### Feature highlights
 
-- **Start your node at system startup (Mac/Windows) and control it from your OS** using the convenient menubar/system tray menu.
+- **Start your node at system startup (Mac/Windows) and control it from your OS** by right-clicking the IPFS icon on the top right of your computer's menubar. <!-- ![IPFS Menubar Icon](.images/ipfs-menubar-icon.png). How do I get an image added? Is this the right format for an inline image? -->
```

Comment:
```suggestion
- **Start your node at system startup (Mac/Windows) and control it from your OS** using the convenient menubar/system tray menu.
```

Again, the location of the IPFS icon is specific to each operating system. We can expect users to know where to look if we say _status bar icon_ or something like that. Revert this back to the original.